pkg/secrethub/credentials/providers.go

```diff
@@ -0,0 +1,102 @@
+package credentials
+
+import (
+	"errors"
+
+	awssdk "github.com/aws/aws-sdk-go/aws"
+
+	"github.com/secrethub/secrethub-go/internals/api"
+	"github.com/secrethub/secrethub-go/internals/auth"
+	"github.com/secrethub/secrethub-go/internals/aws"
+	"github.com/secrethub/secrethub-go/internals/crypto"
+	"github.com/secrethub/secrethub-go/pkg/secrethub/credentials/sessions"
+	"github.com/secrethub/secrethub-go/pkg/secrethub/http"
+)
+
+// Verifier exports verification bytes that can be used to verify signed data is processed by the owner of a signer.
+type Verifier interface {
```

Comment:
Are these interfaces in the right file? They seem a bit lost here.

Reply:
Moved them. But not sure whether they are in the correct place now...